servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkLibraryPlugin.groovy

```diff
@@ -209,6 +210,30 @@ final class ServiceTalkLibraryPlugin extends ServiceTalkCorePlugin {
     }
   }
 
+  private static void applyPitestPlugin(Project project) {
+    project.configure(project) {
+      pluginManager.apply("info.solidsoft.pitest")
+
+      pitest {
+        pitestVersion = project.properties["pitestVersion"]
+        junit5PluginVersion = project.properties["pitestPluginJunit5Version"]
```

Comment:
Oh, I like this style! Didn't know it's possible to access `gradle.properties` from here. We can remove `Versions.groovy` then